akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -174,3 +174,11 @@ and then it will behave as in Akka 2.5.x:
 ```
 akka.coordinated-shutdown.run-by-actor-system-terminate = off
 ```
+
+## Akka Typed API changes
```

Comment:
@patriknw, added this API change notice. I have looked around to see other examples of API change notices for APIs makes as 'may change', but didn't find any. 

I'm not so sure about the wording. So, any feedback is welcome. 

I also added new methods to the list. Maybe not strictly needed, but looks nice to have all the changes listed, IMO.